src/models/armTemplates.ts

```diff
@@ -35,10 +44,18 @@ export interface ArmResourceTemplate {
   variables?: any;
 }
 
+export interface ArmParameters {
+  [key: string]: {
+    type: string;
+    value?: string;
+    defaultValue?: string;
+  };
+}
+
 /**
  * Represents an Azure ARM deployment
  */
 export interface ArmDeployment {
   template: ArmResourceTemplate;
-  parameters: { [key: string]: any };
+  parameters: any;
```

Comment:
It would be, but there are times where parameters object doesn't line up. At times, the type is `{ [key: string]: string}` which then gets normalized to the `ArmParameters` type later on. I can look into updating that